docs/docusaurus/i18n/en.json

```diff
@@ -2005,8 +2006,9 @@
       "Debug": "Debug",
       "Contribute": "Contribute",
       "Technical Reference": "Technical Reference",
-      "Proposals": "Proposals",
+      "Design Documents": "Design Documents",
       "FAQ": "FAQ",
+      "Proposals": "Proposals",
```

Comment:
Why is it necessary to keep this line in this file?

Reply:
This file is auto-generated by write-translations.js.